modules/ROOT/pages/branded_ios_app/publishing_ios_app_10.adoc

```diff
@@ -0,0 +1,88 @@
+= Additional Server Configuration 
+:apple-universal-link-support-url: https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html
+:admin_manual: https://doc.owncloud.com/server/administration_manual/index.html
+:apple-app-site-association-file-url: https://developer.apple.com/documentation/security/password_autofill/setting_up_an_app_s_associated_domains
+:aliasmatch-uri: https://httpd.apache.org/docs/2.4/mod/mod_alias.html#aliasmatch
+:rewritecond-directive-uri: http://httpd.apache.org/docs/2.2/mod/mod_rewrite.html#RewriteCond
+:generate-ios-app-url: branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app
+
+== Add Support for Apple Universal Links
+
+There's some special changes that need to be made. 
+Quoting from Apple's official documentation on {apple-universal-link-support-url}[Universal Link Support]:
+
+[quote]
+____
+Adding support for universal links is easy. 
+There are three steps you need to take:
+
+. Create {apple-app-site-association-file-url}[an apple-app-site-association file] that contains JSON data about the URLs that your app can handle.
+. Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the `.well-known` subdirectory.
+. Prepare your app to handle universal links. 
+____
+
+The `apple-app-site-association` directory is generated by xref:branded_ios_app/publishing_ios_app_7.adoc#generate-ios-app[ownBrander] and must be served statically over HTTPS. 
```

Comment:
The `apple-app-site-association` directory

From that what I read and understood, it is NOT a directory that is generated by ownBrander but a data file.
Pls also see the statement in line 35. (yes a bit confusing but so it works...)

Therefore I propose to write:
The `apple-app-site-association` data

Reply:
Agreed.